### PR TITLE
fix: add tooltips to validators table columns

### LINKED ISSUE #435: What happened to tooltips on the 'See all' validators view?
Screen Shot 2019-09-27 at 1 42 13 PM

What are these strange numbers and what do they mean????

### PR DESCRIPTION
**Pull request by manuel-calavera**
Fixes #435

### CODE REVIEW

**Comment by manuel-calavera** [comment]
@sunnygleason minor change of adding tooltips

File: src/v2/components/Validators/Table/index.jsx [hunk 1]
```diff
@@ -112,10 +113,18 @@ const ValidatorsTable = ({separate}: {separate: boolean}) => {
...
+              </TableCell>
+              <TableCell width={170}>
+                Staked SOL <HelpLink text="" term="" />
+              </TableCell>
+              <TableCell width={190}>
+                Commission <HelpLink text="" term="" />
+              </TableCell>
+              <TableCell width={130}>
+                Uptime <HelpLink text="" term="" />
+              </TableCell>
```
Comment (sunnygleason) [inline src/v2/components/Validators/Table/index.jsx]: @manuel-calavera are these just placeholders?

Comment (manuel-calavera) [reply src/v2/components/Validators/Table/index.jsx]: yes, i will be provided with actual values later

**Pull request merged**